src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -4,27 +4,27 @@
       <v-card flat>
         <!-- Summary Header -->
         <div class="agreement-summary-header">
-            <v-icon>mdi-handshake</v-icon>
-            <label class="agreement-summary-title">
-                <strong>Incorporation Agreement and Benefit Company Articles</strong>
-            </label>
+          <v-icon color="#38598A">mdi-handshake</v-icon>
```

Comment:
If we were to set the icon colour in the base style for v-icons would it inherit throughout? 
I know Vuetify was tricky sometimes about inheritance, if color was a prop for instance.

Reply:
That's clever.

There are many interesting things on the following page but I don't see any about the icons :/  Also we've been using some v-icons with conditional colours, which we would have to keep doing.

https://vuetifyjs.com/en/features/theme/#customizing

However, I am aware of some "style consistency" tickets in the backlog. When we work on them, we should look into this a bit deeper. If the Vuetify theme won't help us directly, I think we should consider a common set of classes, eg "v-icon-check-ok", "v-icon-header", "v-btn-remove", etc.

I can only find a couple of these "style" tickets atm (#5368, #5470) so we'll have to remember to do this ^^ when we work on them.